### ISSUE TITLE
Keyboard accessibility , cannot tabulate  the component  ngx-int-tel-input.

### ISSUE THREAD

**Issue by bouyaahmedsami** [webcat12345/ngx-intl-tel-input#421]
I'm trying to use your library for our project so I referred your demo page.

They said that it is keyboard accessible, however, we cannot tabulate the component,and have access to it with keyborad arrows  .
Do you have any backlog item to make it compliance with accessibility in near future ?
Do you have any tips , or solution to have the component accessible with keyboard ? 

Thank you
